db/crud.go

```diff
@@ -1037,10 +1037,16 @@ func (db *Database) resolveConflict(localDoc *Document, remoteDoc *Document, doc
 	localDocBody := localDoc.GetDeepMutableBody()
 	localDocBody[BodyId] = localDoc.ID
 	localDocBody[BodyRev] = localDoc.SyncData.CurrentRev
+	localDocBody[BodyAttachments] = localDoc.SyncData.Attachments
+	localDocBody[BodyExpiry] = localDoc.SyncData.Expiry
+	localDocBody[BodyDeleted] = localDoc.Deleted
 
 	remoteDocBody := remoteDoc.GetDeepMutableBody()
 	remoteDocBody[BodyId] = remoteDoc.ID
 	remoteDocBody[BodyRev] = remoteDoc.RevID
+	remoteDocBody[BodyAttachments] = remoteDoc.DocAttachments
+	remoteDocBody[BodyExpiry] = remoteDoc.Expiry
+	remoteDocBody[BodyDeleted] = remoteDoc.Deleted
```

Comment:
I'm not sure why remote and local are pulling attachments and expiry from different fields.

Could you explain why here in a comment if it's valid?